stream_alert_cli/package.py

```diff
@@ -175,45 +186,47 @@ def _resolve_third_party(self, temp_package_path):
             LOGGER_CLI.info('No third-party libraries to install.')
             return True
 
-        LOGGER_CLI.info('Installing third-party libraries: %s', ', '.join(third_party_libs))
+        LOGGER_CLI.info(
+            'Installing third-party libraries: %s',
+            ', '.join(third_party_libs))
         pip_command = ['pip', 'install']
         pip_command.extend(third_party_libs)
         pip_command.extend(['--upgrade', '--target', temp_package_path])
 
         # Return True if the pip command is successfully run
-        return run_command(pip_command, cwd=temp_package_path)
+        return run_command(pip_command, cwd=temp_package_path, quiet=True)
 
     def _upload(self, package_path):
-        """Upload the StreamAlert package and sha256 to S3.
+        """Upload the StreamAlert package and sha256 sum to S3.
 
         Args:
             package path [string]:  Full path to the zipped dpeloyment package
 
         Returns:
-            [boolean] Indicating a successful upload
-
-        Raises:
-            Exception if client put_object fails
+            [boolean] Indicating a successful S3 upload
         """
         LOGGER_CLI.info('Uploading StreamAlert package to S3')
         client = boto3.client(
             's3', region_name=self.config['global']['account']['region'])
-        # the zip and the checksum file
+
         for package_file in (package_path, '{}.sha256'.format(package_path)):
             package_name = package_file.split('/')[-1]
             package_fh = open(package_file, 'r')
```

Comment:
we haven't ran into this issue on any conf file loading yet, it seems v unlikely